client/src/components/Settings.js

```diff
@@ -1,40 +1,65 @@
-import React from "react";
+import React, { useState } from "react";
 import NavBar from "./NavBar";
 import Form from "react-bootstrap/Form";
+import axios from "axios";
+import { constants, errors, routes } from "../Common";
+import { ToastsStore } from "react-toasts";
 
-class Account extends React.Component {
-	constructor(props) {
-		super(props);
-	}
+export default function Settings() {
+	const token = localStorage.getItem(constants.TOKEN);
```

Comment:
Can you fetch this at function-call time?

Basically, fetch the token every time the button is clicked instead of storing it as component state